class OutOfRangeError(Exception):
    pass


class AlreadyChosenError(Exception):
    pass

